src/main/java/entities/User.java

```diff
@@ -0,0 +1,133 @@
+package entities;
+
+public class User {
+    // ArrayList of User's favourited posts
+    private ArrayList<Post> favourites;
+
+    // ArrayList of Posts User has made
+    private ArrayList<Post> posts;
+
+    // boolean relating if User has admin privileges
+    private boolean isAdmin;
+
+    // Number of Users created
+    public static int numUsers = 0;
+
+    // id is the unique identifier of User, equal to number of posts at time of creation
```

Comment:
Equal to the number of posts or the number of users at the time of creation?

Reply:
I meant to use user, my bad!